File: BHao.Web/Binding/Models/KorisnikModel.js
(function (bh) {
    var KorisnikModel = function (data) {

        var self = this;

        self.Id = ko.observable(data ? data.Id() : "");
        //self.KorisnickoIme = ko.observable(data ? data.KorisnickoIme() : "").extend({ required: { message: "Morate unijeti korisničko ime!" } });
        self.Email = ko.observable(data ? data.Email() : "").extend({ email: true, required: { message: "Morate unijeti email!" } });
        self.Ime = ko.observable(data ? data.Ime() : "").extend({ required: { message: "Morate unijeti ime!" } });
        self.Prezime = ko.observable(data ? data.Prezime() : "").extend({ required: { message: "Morate unijeti prezime!" } });
        self.Ulica = ko.observable(data ? data.Ulica() : "").extend({ required: { message: "Morate unijeti ulicu!" } });
        self.Broj = ko.observable(data ? data.Broj() : "").extend({ required: { message: "Morate unijeti broj!" } });
        self.GradId = ko.observable(data ? data.GradId() : "").extend({ required: { message: "Morate odabrati grad!" } });
        self.PostanskiBroj = ko.observable(data ? data.PostanskiBroj() : "").extend({ required: { message: "Morate unijeti poštanski broj!"} });
        self.Telefon = ko.observable(data ? data.Telefon() : "")

        ko.validation.registerExtenders();

        self.Uloge = ko.observableArray(data ? data.Uloge() : []).extend({ uloge: { message: "Korisnik ne može biti uposlenik i klijent istovremeno" } });
        self.IsOnemogucen = ko.observable(data ? data.IsOnemogucen() : false);
        self.Lozinka = ko.observable("");
    }
    bh.KorisnikModel = KorisnikModel;
}(window.BHao));

File: BHao.Web/Models/KorisnikModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BHao.Web.Models
{
    public class KorisnikModel
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Ulica { get; set; }
        public string Broj { get; set; }
        public int GradId { get; set; }
        public string PostanskiBroj { get; set; }
        public string Telefon { get; set; }
        public string[] Uloge { get; set; }
        public bool IsOnemogucen { get; set; }
        public string Lozinka { get; set; }
    }
}